ap_src/ap_app/utils/teams_utils.py

```diff
@@ -0,0 +1,46 @@
+# Third-party Libraries
+import hashlib
+import requests
+
+# Environment
+import environ
+
+env = environ.Env()
+environ.Env.read_env()
+
+
+def retrieve_calendar_data(user, sortValue, username=None):
```

Comment:
The `username` parameter was to define different behaviour for simply receiving a username instead of a request. I changed my approach here and just made the function request-agnostic, so only the user instance is needed.

To address this issue, I removed the `username` parameter.

Reply:
Fixed with 66c9df3